src/radius/radius_attr_parse.h

```diff
@@ -0,0 +1,706 @@
+#ifndef RADIUS_ATTR_PARSE_H
+#define RADIUS_ATTR_PARSE_H
+
+#include <stdbool.h>
+
+#define L7_AF_INET           1  
+#define L7_AF_INET6          2
+
+
+
+#define RADIUS_VALUE_LENGTH             253
+
+#define RADIUS_ATTR_TYPE_USER_NAME                   1
```

Comment:
Is this file copy from somewhere or those are type should be an enum instead of defines

Reply:
And many others defines as enums